src/main/java/io/quarkus/gizmo/Gizmo.java

```diff
@@ -8,4 +21,546 @@ public final class Gizmo {
 
     private Gizmo() {
     }
+
+    //================================================================================
+    // utils used to simplify invocation of static/non-static methods of JDK classes
+    // these constructs are not thread-safe and should not be used concurrently
+    //================================================================================
+
+    /**
+     * Generates the bytecode that calls {@link Object#toString()}.
+     * 
+     * @param target
+     * @param obj
+     * @return the result
+     */
+    public static ResultHandle invokeToString(BytecodeCreator target, ResultHandle obj) {
+        return target.invokeVirtualMethod(TO_STRING, obj);
+    }
+
+    /**
+     * Generates the bytecode that calls {@link Object#equals(Object)}.
+     * 
+     * @param target
+     * @param obj
+     * 
+     * @return the result
+     */
+    public static ResultHandle invokeEquals(BytecodeCreator target, ResultHandle obj1, ResultHandle obj2) {
+        return target.invokeVirtualMethod(EQUALS, obj1, obj2);
+    }
+
+    /**
+     * Generates the bytecode that calls {@code System.out.println(obj)}.
+     * 
+     * @param target
+     * @param obj
+     */
+    public static void systemOutPrintln(BytecodeCreator target, ResultHandle obj) {
```

Comment:
If we get sysout, we also need syserr ;)

Reply:
Done.